server/src/services/timeline.service.ts

```diff
@@ -40,7 +41,9 @@ export class TimelineService {
   private async buildTimeBucketOptions(auth: AuthDto, dto: TimeBucketDto): Promise<TimeBucketOptions> {
     const { userId, ...options } = dto;
     let userIds: string[] | undefined = undefined;
-
+    if (dto?.order === AssetOrder.PREFERENCE) {
+      options.order = auth.user.preferedAlbumOrder as unknown as AssetOrder;
```

Comment:
What is wrong with those types so that you need to cast it to `unknown` first?

Reply:
It's because we have 
```ts
export enum AssetOrderPreference {
  ASC = 'asc',
  DESC = 'desc',
}
``` 
and 
```ts
export enum AssetOrder {
  ASC = 'asc',
  DESC = 'desc',
  PREFERENCE = 'preference',
}
```

Reply:
`AssetOrderPreference` is a subset of `AssetOrderPreference` though, right? I don't understand why TS doesn't get this :D

Reply:
Extending an enum would be cool ...